Dockerfile

```diff
@@ -8,8 +8,9 @@ RUN rm /bin/sh && ln -s /bin/bash /bin/sh
 # Install dependencies
 RUN apt-get update && \
 apt-get dist-upgrade -y && \
-apt-get install -y libmysqlclient-dev git python-pip python-dev apache2 apache2-utils libldap2-dev libsasl2-dev nodejs curl
+apt-get install -y libmysqlclient-dev git python-pip python-dev apache2 apache2-utils libldap2-dev libsasl2-dev nodejs curl libssl-dev libffi-dev xmlsec1
 
```

Comment:
Some of these packages have "-dev" in their names.  What does that mean, exactly?  Are they for development purposes _only_?  Will they need to be replaced by something else for production deployment?